### PR TITLE
add-osx-support

### PR DESCRIPTION
**Pull request by maimon33**

### CODE REVIEW

File: packaging/omnibus/config/projects/cloudify-osx.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# cloudify dependencies/components
+dependency "python"
+dependency "pip"
+dependency "cloudify-cli"
+
+# Version manifest file
+dependency "version-manifest"
+
+exclude "**/.git"
+exclude "**/bundler/git"
```
Comment (opencm) [inline packaging/omnibus/config/projects/cloudify-osx.rb]: Do not duplicate files -
if osx ->
  default_root "/opt"
else ->
  default_root /usr/local/opt

File: packaging/omnibus/config/software/cloudify-manager-blueprints.rb [hunk 2]
```diff
@@ -13,8 +14,16 @@
    spaces='\ \ \ \ '
    str_to_replace="/manager_resources_package:/#{str}n;n;n;c#{spaces}default:  #{manager_single_tar_url}"
    cmd="sed -i \"#{str_to_replace}\" /opt/cfy/cloudify-manager-blueprints/inputs/manager-inputs.yaml"
-   command cmd
+   if osx?
```
Comment (opencm) [inline packaging/omnibus/config/software/cloudify-manager-blueprints.rb]: The replacement of single tar have to be done also on osx

File: packaging/omnibus/config/software/cloudify-manager-blueprints.rb [hunk 3]
```diff
@@ -13,8 +14,16 @@
...
+     command "echo 'OSX'"
+   else
+     command cmd
+   end
    str_to_replace="s|.*#manager_resources_package:.*|#manager_resources_package: #{manager_single_tar_url}|g"
    cmd="sed -i \"#{str_to_replace}\" /opt/cfy/cloudify-manager-blueprints/*-inputs.yaml"
-   command cmd
+   if osx?
```
Comment (opencm) [inline packaging/omnibus/config/software/cloudify-manager-blueprints.rb]: The replacement of single tar have to be done also on osx

File: packaging/linux/provision.sh [hunk 4]
```diff
@@ -7,27 +7,47 @@ AWS_ACCESS_KEY=$4
...
-    git checkout ${CORE_BRANCH}
+    git clone https://github.com/cloudify-cosmo/cloudify-cli.git
 fi
+cd cloudify-cli/packaging/omnibus
+#gitTagExists=$(git tag -l $CORE_TAG_NAME)
+#if [ "$CORE_BRANCH" != "master" ]; then
+#    git checkout -b ${CORE_BRANCH} origin/${CORE_BRANCH}
+#else
+#    git checkout ${CORE_BRANCH}
+#fi
```
Comment (geokala) [inline packaging/linux/provision.sh]: Can we please avoid leaving commented code lying around? We can always recover it through git if needed, so leaving it in and commented is just providing meaningless filler for anyone maintaining this later.

**Pull request closed without merging**